cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -171,7 +171,7 @@ public void execute() {
         // Check if the template exists
         if (!getTemplates().contains(template)) {
             CommandUtil.printError(errStream,
-                    "Template not found, use `ballerina add --list` to view available templates.",
+                    "Cannot find this template. Use `ballerina add --list` to view the available templates.",
```

Comment:
Template not found sounds better in this case though. It's similar to how we simply say "file not found" in error messages. The exact context is the following:
```
$ ballerina add mod -t noneexist
ballerina: Template not found, use `ballerina add --list` to view available templates.
```

Reply:
Changed to the following:

"template not found. Use `ballerina add --list` to view the available templates."